createComponent.js

```diff
@@ -2,14 +2,14 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-const inquirer = require('inquirer');
-const fs = require('fs');
+const inquirer = require("inquirer");
+const fs = require("fs");
 
 const copyrightHeader =
-  '/*---------------------------------------------------------------------------------------------\n ' +
-  '* Copyright (c) Bentley Systems, Incorporated. All rights reserved.\n ' +
-  '* See LICENSE.md in the project root for license terms and full copyright notice.\n ' +
-  '*--------------------------------------------------------------------------------------------*/';
+  "/*---------------------------------------------------------------------------------------------\n " +
```

Comment:
What happened with quotes here?

Reply:
it looks regular in my vscode

Reply:
What?
I mean you shouldnt have made any additional changes in here (i.e. replace single quotes with double quotes), so github would show only changes we care.
Can you try to revert that?
Could it be the case, that you dont have `prettier` installed in your vscode?